src/components/container/SurveyDataChart/SurveyDataChart.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect, useRef, useContext } from 'react'
+import { DateRangeContext } from '../../presentational/DateRangeCoordinator/DateRangeCoordinator'
+import { DailyDataProvider, DailyDataQueryResult, checkDailyDataAvailability, queryDailyData } from '../../../helpers/query-daily-data'
+import { add, format, parseISO } from 'date-fns'
+import MyDataHelps, { SurveyAnswer, SurveyAnswersPage } from '@careevolution/mydatahelps-js'
+import getDayKey from '../../../helpers/get-day-key'
+import { WeekStartsOn, getMonthStart, getWeekStart } from '../../../helpers/get-interval-start'
+import DataChart, { AreaChartOptions, BarChartOptions, LineChartOptions } from '../../presentational/DataChart/DataChart'
+
+export interface SurveyAnswerChartLine {
+	label: string;
+	surveyName: string;
+	stepIdentifier: string;
+	resultIdentifier: string;
+}
+
+export interface SurveyDataChartProps {
+    title?: string
+    intervalType?: "Week" | "Month"
+    weekStartsOn?: WeekStartsOn
+    lines: SurveyAnswerChartLine[],
```

Comment:
Seems weird to call these lines when it might be a bar or area chart?

Reply:
Yeah, this is a legacy of bringing it over from a project where I only supported lines, will change.